Flowchart/obj/Debug/netcoreapp1.0/FlowchartCreator.csproj.FileListAbsolute.txt

```diff
@@ -8,3 +8,23 @@ C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\ne
 C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\netcoreapp1.0\FlowchartCreator.pdb
 C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\netcoreapp1.0\UserSecretsAssemblyInfo.cs
 C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\netcoreapp1.0\FlowchartCreator.csprojResolveAssemblyReference.cache
+C:\Users\Sam\Desktop\Flowchart-Creator-production\Flowchart\bin\Debug\netcoreapp1.0\FlowchartCreator.deps.json
```

Comment:
Coolio. I saw [this post](https://stackoverflow.com/questions/588077/what-visual-studio-files-should-be-ignored-by-subversion-to-minimize-conflicts) that says I can just kill off the entire bin and obj directories in git, so I did that. Woo!!

See 016b235 for specifics